user-interface/src/main/java/life/qbic/datamanager/views/general/contact/AutocompleteContactField.java

```diff
@@ -0,0 +1,178 @@
+package life.qbic.datamanager.views.general.contact;
+
+import static java.util.Objects.isNull;
+
+import com.vaadin.flow.component.combobox.ComboBox;
+import com.vaadin.flow.component.customfield.CustomField;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.Span;
+import com.vaadin.flow.data.binder.Binder;
+import com.vaadin.flow.data.renderer.ComponentRenderer;
+import com.vaadin.flow.data.validator.EmailValidator;
+import java.util.ArrayList;
+import java.util.List;
+import life.qbic.datamanager.views.general.HasBinderValidation;
+
+/**
+ * <b>A component for contact person input</b>
+ *
+ * <p>Provides components to add a contact person with a name and email.
+ * Includes methods for basic input validation.</p>
+ *
+ * @since 1.0.0
+ */
+public class AutocompleteContactField extends CustomField<Contact> implements
+    HasBinderValidation<Contact> {
+
+
+  private final ComboBox<Contact> contactSelection;
+
+  private final ComboBox<String> nameField;
+  private final ComboBox<String> emailField;
+  private final Binder<Contact> binder;
+
+  public AutocompleteContactField(String label) {
+    setLabel(label);
+    addClassName("contact-field");
+    binder = new Binder<>();
+    binder.addStatusChangeListener(event -> updateValidationProperty());
+
+    contactSelection = new ComboBox<>();
```

Comment:
NIT: You could add a placeholder to this field as well such as: 
"You can select a contact if it's already in our database"